spec/myst/env_spec.mt

```diff
@@ -0,0 +1,117 @@
+require "stdlib/spec.mt"
+
+initial_env = ENV.to_map
+
+# Keys and values used for testing
+SHAKE_KEY = "Shakespeare"
+SHAKE     = "To be, or not to be: that is the question"
+EIN_STEIN_KEY = "Einstein"
+EIN_STEIN     = "two things are infinite: the universe and human stupidity"
+
+TEST_ENV = { 
+  <SHAKE_KEY>:     SHAKE, 
+  <EIN_STEIN_KEY>: EIN_STEIN, 
+  <"HOME">: "/Users/Bob",
+  <"PWD">:  "/Users/Bob/Projects/myst-lang/myst"
+}
+
+def set_test_env
+  ENV.set(TEST_ENV)
+end
+
+def random_key
+  key = ""
+  16.times {key += Random.rand(10).to_s}
+  key
+end
+
+describe("ENV[]") do
+  it("returns the value associated with a key") do
+    set_test_env
+    assert(ENV[SHAKE_KEY])     == SHAKE
+    assert(ENV[EIN_STEIN_KEY]) == EIN_STEIN
+  end
+
+  it("returns a nil, if no value is set with the given key, otherwise always a string") do
+    set_test_env
+    assert(ENV[SHAKE_KEY]).is_a(String) 
+    assert(ENV[EIN_STEIN_KEY]).is_a(String)
+    assert(ENV[random_key]).is_a(Nil)
+  end
+end
+
+describe("ENV.fetch") do
+  it("is just the same as ENV[] when the key is present in the environment, if not: fetch returns provided default (which defaults to nil) instead of nil; if no default is specified, an error is raised") do
+    set_test_env
+    assert(ENV.fetch(random_key, "none")) == "none"
+    assert(ENV.fetch(SHAKE_KEY)) == SHAKE
+    assert{ENV.fetch(random_key)}.raises
```

Comment:
Spaces around block bodies would make this a little more obvious that the last one is a `BlockAssertion` :)

```ruby
assert{ ENV.fetch(random_key) }.raises
```